src/App.js

```diff
@@ -157,23 +157,24 @@ function App() {
       <section className="grey section">
         <div className="card">
           <h2 style={{textAlign:"center"}}>Look at the animation below to learn how a filter (shown as the red matrix) works on images to calculate each element of the feature map</h2>
+          <p style={{textAlign:"center"}}>(You can also explore our slides on filters (slide 28) from our <a target="_blank" href="https://docs.google.com/presentation/d/1je-mnp1E6y-wj3q_5GNHG8jJm9ckLTyoFNjF3Qj_6LM/edit?usp=sharing" rel="noopener noreferrer">CNN high-school lesson)</a></p> 
+          <p style={{textAlign:"center"}}>Note that whenever the filter hovers over a portion of the image, the corresponding element on the
```

Comment:
If everything in this `<div>` is just `text-align: center;`, you can apply it to the parent to reduce duplication! May also be a good choice to create a `.text-center` class.

Reply:
will push these changes in another PR